src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -27,33 +35,65 @@
      [react/text {:style {:font-size 15 :color colors/blue}}
       (i18n/label :t/get-stickers)]]]])
 
-(defn- stickers-panel [stickers]
-  [react/scroll-view {:style {:flex 1} :condtent-container-style {:flex 1}}
-   [react/view {:style styles/stickers-panel}
-    (for [{:keys [uri] :as sticker} stickers]
-      ^{:key uri}
-      [react/touchable-highlight {:style    {:height 75 :width 75 :margin 5}
-                                  :on-press #(re-frame/dispatch [:chat/send-sticker sticker])}
-       [react/image {:style {:resize-mode :cover :width "100%" :height "100%"} :source {:uri uri}}]])]])
+(defview stickers-panel [stickers window-width]
+  [react/view {:width window-width :flex 1}
+   [react/scroll-view
+    [react/view {:style styles/stickers-panel}
+     (for [{:keys [uri] :as sticker} stickers]
+       ^{:key uri}
+       [react/touchable-highlight {:style    {:height 75 :width 75 :margin 5}
+                                   :on-press #(re-frame/dispatch [:chat/send-sticker sticker])}
+        [react/image {:style {:resize-mode :cover :width "100%" :height "100%"} :source {:uri uri}}]])]]])
```

Comment:
not related to this PR but is the uri a remote source? We might want to start using aggressive caching with https://github.com/DylanVann/react-native-fast-image if that is the case

Reply:
yes that makes a lot of sense, thanks, i'll do it in separate PR